docs/CONFIG.md

```diff
@@ -9243,6 +9308,33 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+## EVM.GasEstimator.DAOracle
+```toml
+[EVM.GasEstimator.DAOracle]
+OracleType = 'OPOracle' # Example
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+CustomGasPriceAPICalldata = '' # Default
+```
+
+
+### OracleType
+```toml
+OracleType = 'OPOracle' # Example
+```
+OracleType refers to the oracle family this config belongs to (OP, Arbitrum, etc).
```

Comment:
Please list all the available Oracle types, otherwise one would have to go through the code to set the right value